addons/stock/models/stock_picking.py

```diff
@@ -431,20 +431,32 @@ def _compute_is_signed(self):
         for picking in self:
             picking.is_signed = picking.signature
 
-    @api.depends('move_lines', 'state', 'picking_type_code', 'move_lines.forecast_availability', 'move_lines.forecast_expected_date')
+    @api.depends('state', 'picking_type_code', 'scheduled_date', 'move_lines', 'move_lines.forecast_availability', 'move_lines.forecast_expected_date')
     def _compute_products_availability(self):
-        self.products_availability = False
-        self.products_availability_state = 'available'
-        pickings = self.filtered(lambda picking: picking.state not in ['cancel', 'draft', 'done'] and picking.picking_type_code == 'outgoing')
-        pickings.products_availability = _('Available')
+        pickings = self.filtered(lambda picking: picking.state in ('waiting', 'confirmed', 'assigned') and picking.picking_type_code == 'outgoing')
```

Comment:
Tu devrais pas utiliser ton _consuming_picking_types

Reply:
it is on stock.move model and a picking cannot have a manufacturing type.